pkg/sbom/implementation.go

```diff
@@ -75,11 +75,12 @@ func encodeInvalidRune(r rune) string {
 	return "C" + strconv.Itoa(int(r))
 }
 
-// checkEnvironment returns a bool indicating if Spec's Path exists. If the path
-// does not exist, it returns false and a nil error. If an error occurs while
-// checking the directory, it returns false and the error.
-func checkEnvironment(spec *Spec) (bool, error) {
-	dirPath, err := filepath.Abs(spec.Path)
+// checkEnvironment returns a bool indicating if the specified path for an APK's
+// filesystem exists. If the path does not exist, it returns false and a nil
+// error. If an error occurs while checking the directory, it returns false and
+// the error.
+func checkEnvironment(apkFSPath string) (bool, error) {
```

Comment:
super nit and certainly not your doing, but checkEnvironment seems a bit confusing vs. checkPathExists or something like that. Environment has so many loaded meanings.

Reply:
I agree, the name is vague and unhelpful — time to change it...